packages/renderer/src/lib/ui/Typeahead.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
```

Comment:
Does this PR is suppose to include the Typeahead component ? Giving the title I thought it would be backend only

Reply:
I created the branch based on the Typeahead's branch, to be able to test the typeahead of private images with the fix. I'll rebase it to main